packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -18,11 +19,16 @@ const TimestampContainer = ({
   suffix,
   timezone,
   script,
+  locale,
 }) => {
   if (!isValidDateTime(new Date(timestamp))) {
     return null;
   }
 
+  if (locale) {
+    moment.locale(locale);
```

Comment:
Have you considered passing the locale into `formatUnixTimestamp` and only setting the moment locale locally? http://momentjs.com/docs/#/i18n/instance-locale/

Reply:
Thats a good idea. 👍